docs/_ext/djangodocs.py

```diff
@@ -109,8 +109,9 @@ def warner(msg):
                                                    warn=warner,
                                                    linenos=linenos,
                                                    **highlight_args)
+    css_classes = ['highlight-%s' % lang, 'clippify']
```

Comment:
I'm inclined to use a more generic CSS class name like "snippet" that might be useful for other purposes. What do you think?